x-pack/plugins/security_solution/public/common/lib/local_storage/types.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export interface SecuritySolutionStorage {
+  getAllTimelines: () => object | null;
```

Comment:
I think we can define this better since we know the type of timeline maybe something like that `Record<TimelineId, TimelineModel>`

Reply:
and TmelineId can be a string or more strict to match all the type fo the timeline ids